protocols/Newport_scientific/LCMS_bioassay_part1.ot2.py

```diff
@@ -0,0 +1,77 @@
+from opentrons import labware, instruments
+
+# customization
+sample_num = 96
+reagent_row_1 = 6
+
+incubation_plate = labware.load('96-PCR-tall', 1)
```

Comment:
You are technically _allowed_ to use numbers instead of strings but we are trying to make it best practice to use strings. Pls change :)